www/configurator/src/app/css-transpiler/css-transpiler.js

```diff
@@ -30,15 +32,30 @@ class CssTranspiler {
   constructor(css, cssVars) {
     this.templateCss = css;
     this.templateCssVars = cssVars;
+    this.asyncPostcssWorker = {
+      worker: new Worker(),
+      didRespond: true
+    };
   }
 
   /**
    * Function to retranspile page css with new css vars, using postcss
    * @param {Object} passedCssVars - Json object, where the key represents the css variable to be modified,
    *    and the value represents the variables new value
-   * @returns {string} - the newly transpiled page css with varibale values
+   * @returns {Promise} - Will resolve with the response from a webworker, containing the newly transpiled css
    */
   getCssWithVars(passedCssVars) {
+    // First check if we have a worker, and if it didRespond
+    if (this.asyncPostcssWorker.worker && !this.asyncPostcssWorker.didRespond) {
+      // Terminate the worker, and recreate it
+      this.asyncPostcssWorker.worker.terminate();
+      this.asyncPostcssWorker = {
+        worker: new Worker(),
```

Comment:
The filename is being passed on about line 25, when we Define/require the Worker.

The `worker-loader!` tells the web pack to use the worker-loader, and to use the filepath that is following it.